source/developer/dev-setup-compiling.rst

```diff
@@ -16,7 +16,7 @@ Mattermost uses `GNU Make <https://www.gnu.org/software/make/>`_ to control the
 
   ``cd ~/go/src/github.com/mattermost/platform``
 
-The first time that you use the ``make run`` command can take a substantial amount of time depending on your machine's processor speed and memory size, and on the speed of your network. If you are developing in a VM, make sure that the VM has at least 2G of memory assigned to it.
+The first time that you use the ``make run`` command can take a substantial amount of time depending on your machine's processor speed and memory size, and on the speed of your network. If you are developing in a VM, make sure that the VM has at least 2G of memory assigned to it. Refer to the :doc:`Developer Flow <developer-flow>` to start contributing.
```

Comment:
Can you change the sentence a bit so that the "start contributing" is at the beginning? Like this...
```
To start contributing to Mattermost, see :doc:`developer-flow`.
```
Also, so that it stands out more, can you put it on its own line?